rllib/algorithms/dqn/torch/dqn_rainbow_torch_rl_module.py

```diff
@@ -0,0 +1,319 @@
+from typing import Dict, List, Tuple, Union
+
+from ray.rllib.algorithms.dqn.dqn_rainbow_rl_module import (
+    DQNRainbowRLModule,
+    ATOMS,
+    QF_LOGITS,
+    QF_PROBS,
+    QF_TARGET_NEXT_PREDS,
+    QF_TARGET_NEXT_PROBS,
+)
+from ray.rllib.algorithms.sac.sac_rl_module import QF_PREDS
```

Comment:
@sven1977 Should we import here from `SAC` or should we define in each algorithms their own keys and risk duplicates across the whole code base?

Reply:
Already answered [here](https://github.com/ray-project/ray/pull/43196#discussion_r1495814199).